src/main/cpp/mechanisms/ClimberManager/decoratormods/InitializeState.cpp

```diff
@@ -70,5 +70,5 @@ bool InitializeState::IsTransitionCondition(bool considerGamepadTransitions)
 {
 	// To get the current state use m_mechanism->GetCurrentState()
 
-	return (m_mechanism->IsEnabled());
+	return (m_mechanism->IsEnabled() && false);
```

Comment:
Is this a permanent change?

Reply:
No, just until Monday night, for auto testing purpose.